examples/README.md

```diff
@@ -4,56 +4,7 @@ A set of examples showcasing how to use rust-libp2p.
 
 ## Getting started
 
-- [Ping](ping.rs)
-
-  Small `ping` clone, sending a ping to a peer, expecting a pong as a response. See
-  [tutorial](../src/tutorials/ping.rs) for a step-by-step guide building the example.
 
 ## Individual libp2p features
 
-- [Chat](./chat.rs)
-
-   A basic chat application demonstrating libp2p and the mDNS and floodsub protocols.
-
-    - [Gossipsub chat](./gossipsub-chat.rs)
-
-      Same as the chat example but using mDNS and the Gossipsub protocol.
-
-    - [Tokio based chat](./chat-tokio.rs)
-
-      Same as the chat example but using tokio for all asynchronous tasks and I/O.
```

Comment:
One thing to tidy up here could be:

- Replace current chat example with gossipsub. It is preferred over floodsub.
- Remove the tokio chat example. I'd rather just convert one of the others to use tokio instead of having a dedicated clone of an example just with another executor.